local-cli/default.config.js

```diff
@@ -41,4 +33,16 @@ var config = {
   }
 };
 
+function getRoots() {
```

Comment:
It's not necessary because the function is not part of the object that is exported :)